packages/cxl-ui/scss/global/cxl-adeft-accordion.scss

```diff
@@ -1,56 +1,50 @@
 @use "~@conversionxl/cxl-lumo-styles/scss/mq";
 @use "../mixins";
 
-$bg-color: #0000ff;
 $standard-margin: 16px;
 $medium-margin: 24px;
 $large-margin: 32px;
```

Comment:
Can drop. These are provided at https://cdn.vaadin.com/vaadin-lumo-styles/1.6.0/demo/sizing-and-spacing.html

Reply:
@lkraav fixed in the latest commit